resources/com.sap.piper/pipeline/stashSettings.yml

```diff
@@ -6,13 +6,23 @@ Init:
     excludes: ".pipeline/**"
 
 'Pull-Request Voting':
-  unstash: ['source']
+  unstash:
+  - source
   stashes: []
 
-'Build':
-  unstash: ['source']
+Build:
+  unstash:
+  - source
   stashes: []
 
-'Acceptance':
-  unstash: ['buildResult']
+Acceptance:
+  unstash:
+  - buildResult
+  - deployDescriptor
+  stashes: []
+
+Release:
```

Comment:
I guess this is the important part for fixing the issue?

Reply:
yepp